k8s/namespaces/fees-pay/fees-register-api/perftest.yaml

```diff
@@ -6,6 +6,6 @@ spec:
   releaseName: fees-register-api
   values:
     java:
-      image: hmctspublic.azurecr.io/fees-register/api:prod-bd2eecc-20220503191941 #{"$imagepolicy": "flux-system:fees-register-api"}
+      image: hmctspublic.azurecr.io/fees-register/api:prod-bd2eecc-20220503191941 #{"$imagepolicy": "flux-system:perftest-fees-register-api"}
```

Comment:
Just checking, this will update the image to pr-454-* that ok?

Reply:
Hi Thomas, yes that is correct. We want the same PR from demo into perftest